docs/source/conf.py

```diff
@@ -112,6 +112,7 @@ def setup(_):
             "examples/pipeline/*.py",
             "examples/db_connector/*.py",
             "examples/generics/*.py",
+            "examples/telegram/interfaces/*.py",
```

Comment:
Add similar entry to `regenerate_apiref`: `("dff.connectors.messenger.telegram", "telegram_interface")` and replace `("dff.connectors.messenger", "messenger_interfaces")` with something like `("dff.connectors.messenger.generics", "common_messenger_interface")`

Reply:
resolved